GOVERNANCE.md

```diff
@@ -0,0 +1,123 @@
+# CoreDash Project Governance - WIP DRAFT
+
+**Important note**: This governance document is not finalized or implemented at
+this time, since we are still in the early days of forming the project. This
+document provides people with an opportunity to provide feedback and sets some
+initial expectations about the type of governance you can expect coming soon.
+
+The CoreDash umbrella project is composed as a federation of individual projects,
+some independent and some interdependent, each of which focuses on some aspect
+of CoreDash. Our governance reflects this federated structure.
+
+CoreDash is a Linux Foundation project.
+
+## Individual Subproject Governance
```

Comment:
Does it mean that to be considered as a sub project of CoreDash, it will have to add these files (CODE_OF_CONDUCT.md, GOVERNANCE.md) ?

Reply:
For subprojects, I would recommend linking to the CODE_OF_CONDUCT.md for CoreDash to avoid having copies floating around if we need to change anything (like the names of the people who take the reports). I've created a templates directory and added something that you can copy to your project after we announce CoreDash.

I've also added a template for governance. One of the things that I really like about the umbrella project / subproject governance structure is that each subproject still operates independently. Subprojects can have different ways to select new maintainers and make decisions, so template can be completely customized for your needs. This gives each subproject quite a bit of freedom while also being part of the overall project, and there is a link at the bottom to the CoreDash governance. Have a look at the governance template, and I think you'll see what I mean. If not, I'm happy to answer questions.

Reply:
> I would recommend linking to the CODE_OF_CONDUCT.md for CoreDash

Makes sense, we also do this in Prometheus with the CNCF CoC: https://github.com/prometheus/prometheus/blob/main/CODE_OF_CONDUCT.md

Reply:
Thanks for the answers :)